core/modules/layout/layout.module

```diff
@@ -339,6 +339,14 @@ function layout_menu() {
     'file' => 'layout.context.admin.inc',
   ) + $base;
 
+  $items['admin/structure/layouts/manage/%layout_tempstore/relationship/add'] = array(
+    'title' => 'Add context relationship',
```

Comment:
The label `Add relationship` is probably sufficient here.

Reply:
Yeah, I agree. It's already under a second that says "Contexts", so the additional wording of "Context Relationship" is redundant.